<launch>

	<param name="cluedo_path_rooms_file" type="string" value="$(find robocluedo)/config/cluedo_rooms.txt" />
	<!-- <node pkg="turtlesim" name="turtlesim" type="turtlesim_node" /> -->
	<node pkg="robocluedo" name="cluedo_random_room" type="cluedo_random_room" output="screen"></node>
	
</launch>
